src/modules/layerManager/components/LayerManager.js

```diff
@@ -178,9 +178,9 @@ export class LayerManager extends MvuElement {
 			<style>
 				${css}
 			</style>
-			<div class="layermanager overflow-container">
+			<div class="layermanager overflow-container draggable">
 				<div class="title">${translate('layerManager_title')}</div>
-				<ul class="layers">
+				<ul class="layers draggable">
```

Comment:
the draggable attribute in line 189 should be sufficient, for a full drag&drop-funcitonality of the layers, or not?

I've tested with Chrome and FF so far